### DISCUSSION TITLE
Does bleak support logging to file?

### DISCUSSION THREAD

**Discussion by snowuyl** [hbldh/bleak#838] [Q&A]
* bleak version: 0.15.0a1
* Python version: 3.10.4
* Operating System: Windows 10

### Description
Since bleak print battery level information on terminal when I use bleak to query battery level from Bluetooth device. But I would like to logging battery level info file instead terminal.  Does bleak support logging to file?

**Comment by dlech** [comment]
Bleak is a library and does not dictate what you do with the information read from a device. You can use the Python standard library to write to a file.
